infrastructure/foreman/katello.py

```diff
@@ -241,7 +241,7 @@ def manifest(self, params):
             e = get_exception()
             
             if "Import is the same as existing data" in e.message:
-                return True
+                return False
```

Comment:
This was actually another change that has nothing to do with repository_sets, my bad :)

In the current behavior, even though the manifest does not change, ansible reports in the play recap that task did change . By returning False when the response indicates that there was no change in the manifest data we get that the task did not change.

```
PLAY RECAP ***************
x.x.x.x             : ok=1   changed=0    unreachable=0    failed=0
```

Instead of

```
PLAY RECAP ***************
x.x.x.x             : ok=0   changed=1    unreachable=0    failed=0
```

I had originally intended to modify the module to search and compare most entities prior to attempting to update/create them, but this case was handled with this simpler change.